tgui/packages/common/string.js

```diff
@@ -175,7 +175,7 @@ export const decodeHtmlEntities = (str) => {
     // Basic entities
     .replace(translate_re, (match, entity) => translate[entity])
     // Decimal entities
-    .replace(/&#?([0-9]+);/gi, (match, numStr) => {
+    .replace(/&#?([0-9]+);?/gi, (match, numStr) => {
```

Comment:
Semicolon is required for an html entity to be valid, otherwise it's treated as text.

Reply:
https://www.w3.org/TR/html4/sgml/entities.html

Reply:
Good, I'll revert this. I felt hesitant asking this conditional. The problem seems to be in the extra layer of sanitization as mentioned by @Mothblocks